hw/top_earlgrey/data/top_earlgrey.hjson

```diff
@@ -288,13 +288,15 @@
       type: "usbdev",
       clock_srcs: {clk_i: "io", clk_usb_48mhz_i: "usb"},
       clock_group: "peri",
+      clock_reset_export: ["ast"],
```

Comment:
would it maybe make sense to add this `clock_reset_export` attribute to the reset node definitions further above instead of the peripheral instantiations?

Reply:
yeah the problem is ast right now is doing this on a per module basis.  So if they talk to usb, they want a usb clock/rst, if they talk to adc .. so on and so forth.

The reset group attribute doesn't really make it clear who is related to who.. so i thought it was cleaner to feed it through after the modules, even if it ends up being a lot of duplication.

Reply:
ah, got it (I thought per reset/clk node would be sufficient). never mind then.